nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -557,6 +557,107 @@ public void genericFunctionReturnTypeMultipleReturnStatementsIfElseBlock() {
         .doTest();
   }
 
+  @Test
+  public void genericsChecksForTernaryOperator() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "static class A<T extends @Nullable Object> { }",
+            "  static A<String> testPositive(boolean t) {",
+            "    // BUG: Diagnostic contains: Conditional expression",
```

Comment:
Can we at least use "Conditional expression must have type" as the match string here, just matching on "Conditional expression" is very ambiguous unless someone has in mind the error's full text already.